build.gradle.kts

```diff
@@ -89,28 +94,29 @@ subprojects {
 
   val checkstyleConfig: Configuration by configurations.creating
   dependencies {
-    checkstyleConfig(Dependencies.checkStyle) {
+    checkstyleConfig("com.puppycrawl.tools:checkstyle:9.2") {
```

Comment:
Can move into an afterEvaluate to support?

Reply:
Yeah.
```kotlin
  afterEvaluate {
    val checkstyleConfig: Configuration by configurations.creating
    dependencies {
      checkstyleConfig(libs.checkStyle) {
        isTransitive = false
      }
    }
    ...
  }
```